tests/regressiontests/make/tests/tests-base.m4

```diff
@@ -146,6 +146,14 @@ ADD_TEST_BASH([test-simple], [test_simple_body
 	$< pos -o 'uf ta' | grep -q 'OPT_S=uf ta,POS_S=pos,'
```

Comment:
```suggestion
	$< pos -o 'uf ta' | grep -q 'OPT_S=uf ta,POS_S=pos,'
	$< pos -o 'uf ta' --print-optionals | grep -q '_supplied_arg_prefix=1,_supplied_arg_print_optionals=1,_supplied_arg_la=x'
```

IOW you can add more lines to this test to check whether the expected supplied arguments match or not to what was actually passed.